judgments/views/detail.py

```diff
@@ -123,7 +123,7 @@ def detail(request, document_uri):
 def detail_xml(_request, document_uri):
     document = get_published_document_by_uri(document_uri)
 
-    document_xml = document.content_as_xml()
+    document_xml = document.content_as_xml
```

Comment:
Are you going to update the `detail` view as part of this PR on a followup?

In particular, we can replace the conditional
```
    press_summary_suffix = "/press-summary/1"
    if document_uri.endswith(press_summary_suffix):
```
With the usage of `get_document_from_uri` and then check the type of the returned instance.

and we can now use 

`context["judgment_ncn"] = document.neutral_citation` even for press summaries.

Reply:
Yeah, I wanted to separate those things out.